.gitignore

```diff
@@ -46,3 +46,4 @@ tests/functional/assets
 Dockerfile-*
 root_files/sitemap.xml
 root_files/default-urls.json
+package-lock.json
```

Comment:
Nit: Please ensure the file has an empty new line at the end.

Reply:
@schalkneethling okay I added a new line.